fix_test_clippy.py

```
@@ -0,0 +1,13 @@
+import re
+
+with open('crates/logfwd-io/tests/it/framed_state_machine.rs', 'r') as f:
```

Ruff (0.15.10)

[warning] Unnecessary mode argument

Remove mode argument

(UP015)